_i18n/en/resources/user-guides/join-monero-matrix.md

```diff
@@ -0,0 +1,34 @@
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+The Monero community primarily communicates via IRC channels. We utilize these channels using the Matrix network. An open source network for secure, decentralized communication.
+
+In order to access the Matrix protocol you will need to use a Matrix client. The client we recommend is Element. Element is a free, open-source program that can be accessed via web, desktop, and mobile.
```

Comment:
better avoid using "We", as that warrants the question: we who? and it's hard to answer to that question :)